cron/data/publisher.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2021 Security Scorecard Authors
```

Comment:
publisher*.go seems weird to go in data/ dir, maybe something else?

Reply:
Done. Moved to pubsub/